draft-demarco-oauth-status-attestations.md

```diff
@@ -403,24 +403,55 @@ Credential Issuer MUST respond with an HTTP Response with the status
 code set to 200 and the `status_assertion_responses` array with the related
 Status Assertion Error object.
 
+Below a non-normative example of a Status Assertion Error object in JWT format, with the headers and payload represented in JSON and without applying the signature.
+
+~~~
+{
+    "alg": "ES256",
+    "typ": "status-assertion-error+jwt",
+    "kid": "Issuer-JWK-KID"
+}
+.
+{
+    "iss": "https://issuer.example.org",
+    "jti": "6f204f7e-e453-4dfd-814e-9d155319408c"
+    "credential_hash": $CREDENTIAL-HASH,
+    "credential_hash_alg": "sha-256",
+    "error": "credential_revoked",
+    "error_description": "Credential is revoked."
+    }
+}
+~~~
+
 The Status Assertion Error object must contain the parameters described in the
 table below:
 
 | Header | Description | Reference |
 | --- | --- | --- |
-| **typ** | Depending on the related Status Assertion Request object format, it MUST be set to `status-attestation-error+jwt` or `status-attestation-error+cwt`. | {{RFC7516}} Section 4.1.1 |
-| **alg** | It MUST set to `none`. | {{RFC7516}} Section 4.1.1 |
+| **typ** | REQUIRED. Depending on the related Status Assertion Request object format, it MUST be set to `status-attestation-error+jwt` or `status-attestation-error+cwt`. | {{RFC7516}} Section 4.1.1 |
+| **alg** | REQUIRED. For the Status Assertion Error that do not need to be signed the `alg` value MUST be set to `none`. | {{RFC7516}} Section 4.1.1 |
```

Comment:
: (

MUST use alg none ....

Reply:
There may be situations where an adversary may send hundreds of fake status assertion request objects forcing the issuer to sign hundreds of Status Assertion Errors.

this would consume computational resources and in some cases, depending by the implementation and deployment of the issuer, may cause a resource exhaustion attack

I would change it from MUST to SHOULD if this relaxes our concerns @OR13